### PR TITLE
Switch to 'make bin' instead of 'make binary'

### PR DESCRIPTION
**Pull request by cdrage**
I keep mistyping this when creating the binary as per other projects
that use it commonly (it's usually `make bin` instead of `make binary`).

### CODE REVIEW

**Comment by cdrage** [comment]
@sebgoa do we really need to put a license header in *every* file? Not even Docker has that (and it's pretty much the largest Go project out there: https://github.com/docker/docker/blob/master/Makefile)

**Comment by cdrage** [comment]
@sebgoa 
Updated the PR and to answer your questions:
make already builds binary by default
binary-cross switched to just cross
yeah... validation tests should be added, i'll add this once gofmt + go lint has been added here: https://github.com/kubernetes-incubator/kompose/pull/259/files

**Comment by kadel** [comment]
> yeah... validation tests should be added, i'll add this once gofmt + go lint has been added here: https://github.com/kubernetes-incubator/kompose/pull/259/files

added ;-)

**Comment by cdrage** [comment]
@kadel thanks :)

@sebgoa updated with *all* your suggestions now! Good for review :+1:

**Comment by kadel** [comment]
@cdrage dont' forget to update [.travis.yml](https://github.com/kubernetes-incubator/kompose/blob/master/.travis.yml)

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ $ CGO_ENABLED=0 GO15VENDOREXPERIMENT=1 go build -o kompose main.go
 To create a multi-platform binary, use the `binary-cross` command via `make`:
 
 ```console
-$ make binary-cross
+$ make bin-cross
```
Comment (sebgoa) [inline README.md]: the Makefile uses the `cross` target not the `bin-cross` ?

**Comment by sebgoa** [comment]
@cdrage and yes we need the license header. I don't care what Docker does. Any file that does not have the license header is not properly licensed.. that's how you apply an ASL license throughout your code base.

**Comment by cdrage** [comment]
@sebgoa done. added license and updated readme.

**Comment by sebgoa** [comment]
well, lots of travis errors

**Comment by cdrage** [comment]
@sebgoa Travis passes now. Forgot to update `.travis.yml`

**Pull request merged**